src/common/mailing.ts

```diff
@@ -91,3 +92,95 @@ export const getCommentedAuthorMailParams = (
     },
   };
 };
+
+// taken from sendgrid library itself
+type HttpMethod =
```

Comment:
The type-safety of sendgrid shouts when sending the parameters without casting the method to this type.